client/my-sites/stats/activity-log-search/index.jsx

```diff
@@ -0,0 +1,168 @@
+		event.preventDefault();
+		this.setState( { activeFilter: null } );
+	};
+
+	activeFilterHeaderText() {
+		const { translate } = this.props;
+		switch ( this.state.activeFilter ) {
+			case 'group':
+				return translate( 'Search by Activity Group' );
+			case 'time':
+				return translate( 'Search by time' );
+		}
+	}
+
+	render() {
+		const { translate, filter, siteId } = this.props;
+
+		return (
+			<section className="activity-log-search">
+				{ this.hasTokens() && <ActivityLogSearchTokens filter={ filter } siteId={ siteId } /> }
+				<div className="activity-log-search__filters">
+					{ this.state.activeFilter ? (
+						<div className="activity-log-search__filters-header">
+							<Button
+								className="activity-log-search__filters-back"
+								onClick={ this.handleBackClick }
+								borderless={ true }
+								compact={ true }
+							>
+								<Gridicon icon="arrow-left" />
+								<span>{ translate( 'Go Back' ) }</span>
+							</Button>
+							<div className="activity-log-search__filters-header-text">
+								{ this.activeFilterHeaderText() }
+							</div>
+						</div>
+					) : (
+						<div className="activity-log-search__filters-header">
+							<div className="activity-log-search__filters-header-text">
+								{ translate( 'Search by' ) }
+							</div>
+						</div>
+					) }
+
+					{ 'group' === this.state.activeFilter && (
+						<div className="activity-log-search__filters-groups">
+							<button>
+								<Gridicon
+									icon="posts"
+									className="activity-log-search__filter-group-icon"
+									size={ 18 }
+									data-group="post"
+								/>
+								<span>{ translate( 'Posts and Pages' ) }</span>
+							</button>
+							<button>
+								<Gridicon
+									icon="multiple-users"
+									className="activity-log-search__filter-group-icon"
+									size={ 18 }
+									data-group="user"
+								/>
+								<span>{ translate( 'Users' ) }</span>
+							</button>
+							<button>
+								<Gridicon
+									icon="comment"
+									className="activity-log-search__filter-group-icon"
+									size={ 18 }
+									data-group="comment"
+								/>
+								<span>{ translate( 'Comments' ) }</span>
+							</button>
+							<button>
+								<Gridicon
+									icon="attachment"
+									className="activity-log-search__filter-group-icon"
+									size={ 18 }
+									data-group="attachment"
+								/>
+								<span>{ translate( 'Media and Attachments' ) }</span>
+							</button>
+							<button>
+								<Gridicon
+									icon="cog"
+									className="activity-log-search__filter-group-icon"
+									size={ 18 }
+									data-group="setting"
+								/>
+								<span>{ translate( 'Settings' ) }</span>
+							</button>
+							<button>
+								<Gridicon
+									icon="cog"
+									className="activity-log-search__filter-group-icon"
+									size={ 18 }
+									data-group="term"
+								/>
+								<span>{ translate( 'Categories and Tags' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **92** times:
`translate( 'Categories and Tags Converter' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/activity-log-search-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).